walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/VerifiedIdRequester.kt

```diff
@@ -51,4 +54,13 @@ object VerifiedIdRequester {
             )
         }
     }
+
+    internal suspend fun sendIssuanceCallbackViaCredentialMetadata(
```

Comment:
Can we reuse the existing method instead of creating a new one? The logging is the only difference which is good to have in the old method as well.

Reply:
Done! :)